lib/client.c

```diff
@@ -1853,17 +1852,14 @@ SF_STATUS STDCALL _snowflake_execute_ex(SF_STMT *sfstmt,
     if (request(sfstmt->connection, &resp, queryURL, url_params,
                 url_paramSize , s_body, NULL,
                 POST_REQUEST_TYPE, &sfstmt->error, is_put_get_command)) {
-        s_resp = snowflake_cJSON_Print(resp);
```

Comment:
This changes the behavior, no? Now we don't log `s_resp` unless there is a result capture buffer.

Reply:
Yes this does change the behavior. So is it necessary for us to always log `s_resp`?

Reply:
@sfc-gh-kwagner ? @sfc-gh-vreddy ? What do you think?

Reply:
Yea we should keep it to always log the response when logging in trace mode. Trace mode shouldn't be enabled unless there is an issue and we need a customer to collect more info for us to fix a bug.

Reply:
Makes sense. I'll change it back.

Reply:
Yes I agree. I would prefer to make this modification in a separate PR, so that modifications of this PR is more clear.
I created a new JIRA to track this improvement: SNOW-294232.